assets/glyphs/icon-close.svg

```diff
@@ -1 +1,3 @@
-<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m20 4-16 16m0-16 16 16" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></svg>
\ No newline at end of file
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Remove `icon-` prefix from file name.

Reply:
Are you sure? You added this one. It's only showing up here because I ran `svgo` to optimize the `svg`s.

Reply:
Ah my bad. I probably added those files long time ago. I'll file a ticket to update old file names then.

Side note: try not to update files / code that are outside of the ticket scope. It's hard for reviewer to tell whether the file was changed intentionally or just optimized.

Reply:
ya, sorry...I usually just optimize the entire svg folder.